src/ray/core_worker/actor_handle.h

```diff
@@ -2,12 +2,16 @@
 #define RAY_CORE_WORKER_ACTOR_HANDLE_H
 
 #include <gtest/gtest_prod.h>
+#include "absl/types/optional.h"
 
 #include "ray/common/id.h"
 #include "ray/common/task/task_util.h"
 #include "ray/core_worker/common.h"
 #include "ray/core_worker/context.h"
+#include "ray/gcs/tables.h"
```

Comment:
Instead of storing ActorTableData can we store the relevant fields instead? Basically it would nice if someone reading this code didn't need to know what a GCS is.